example/conduit/t8forest_to_conduit_mesh.cxx

```diff
@@ -0,0 +1,429 @@
+  int                 num_corners;
+
+  /* Here we construct the mesh node object, s.t. it is an explicit, unstructured mesh
+   * with mixed element types. The shape map contains the corresponding element int numbers
+   * for every element type. 
+   */
+
+  Node                mesh;
+  mesh["coordsets/coords/type"] = "explicit";
+  mesh["topologies/mesh/type"] = "unstructured";
+  mesh["topologies/mesh/coordset"] = "coords";
+  mesh["topologies/mesh/elements/shape"] = "mixed";
+  mesh["topologies/mesh/elements/shape_map/point"] = 1;
+  mesh["topologies/mesh/elements/shape_map/line"] = 3;
+  mesh["topologies/mesh/elements/shape_map/tri"] = 5;
+  mesh["topologies/mesh/elements/shape_map/quad"] = 9;
+  mesh["topologies/mesh/elements/shape_map/tet"] = 10;
+  mesh["topologies/mesh/elements/shape_map/hex"] = 12;
+
+  /* 
+   * Furthermore, we need
+   * under topologies/topo/elements/shapes : (shapes array), which
+   * contains the int number of each element in the mesh,
+   * under topologies/topo/elements/sizes : (sizes array), which
+   * contains the sizes of those elements,
+   * topologies/topo/elements/offsets : (offsets array), which
+   * contains the offset, meaning the index at which each element starts in the points array
+   * and topologies/topo/elements/connectivity : (connectivity array), which
+   * contains the points ids that make up an element for each element. 
+   * We need to allocate a num points long array for all but the connectivity array.
+   * Thus we first calculate the number of points.
+   */
+
+  num_elements = t8_forest_get_local_num_elements (forest);
+  int                 connec_iter = 0;
+
+  /* 
+   * Additionally, for the connectivity array we need to allocate the sum
+   * of the number of points over all elements in the mesh. We store this in 
+   * the connec_iter. May need a better name. 
+   */
+
+  for (itree = 0; itree < t8_forest_get_num_local_trees (forest); itree++) {
+/* 
+ * We get the current tree, the scheme for this tree
+ * and the number of elements in this tree. We need the vertices of
+ * the tree to get the coordinates of the elements later. We need
+ * the number of elements in this tree to iterate over all of them.
+ */
+    t8_eclass_scheme_c *scheme =
+      t8_forest_get_eclass_scheme (forest, t8_forest_get_tree_class (forest,
+                                                                     itree));
+    t8_locidx_t         elems_in_tree =
+      t8_forest_get_tree_num_elements (forest, itree);
+    /* We iterate over all elements in the tree */
+    /* Compute the global tree id */
+    gtreeid = t8_forest_global_tree_id (forest, itree);
+    for (ielement = 0; ielement < elems_in_tree; ielement++) {
+      t8_element_t       *element =
+        t8_forest_get_element_in_tree (forest, itree, ielement);
+      T8_ASSERT (element != NULL);
+      t8_element_shape_t  element_shape = scheme->t8_element_shape (element);
+      if (element_shape == T8_ECLASS_PRISM) {
```

Comment:
Could we implement prisms and pyramids as polyhedra, if they don't add support for these elements soon?